src/content/en/2020/mobile-web.md

```diff
@@ -407,6 +412,8 @@ At the start of [the User Journey](#the-user-journey) we mentioned that mobile r
 
 Hopefully by now we have gained a better understanding to reason about this, for instance reasons may include ease of finding and consuming content, the ease of typing, form filling etc.
 
-For larger sites, it’s often not a question of whether to invest in mobile web OR desktop, as they both often complement each other.
+For larger sites, it's often not a question of whether to invest in mobile web OR desktop, as they both often complement each other.
 
 It helps to consider all the four phases of the user journey to understand the full spectrum of opportunities for engaging the user, as well as the risks and challenges in each phase of the journey.
+
+{# TODO(authors): Overall conclusion. #}
```

Comment:
@spanicker There isn't an overall conclusion to this chapter. It should have one for consistency.

Reply:
I've added the following conclusion in an effort to close this out:

> ## Conclusion
> 
> Mobile is now the predominant way of accessing the web, and access to the web has become all the more important in the last year. The needs of mobile are different from those of desktop. Image sizes can, and should, be smaller on mobile due to smaller screens and often limited network but seeing two fifths of improperly sized images show we still have some way to go. Similarly tap targets need to be bigger on mobile since we don;'t have the precision of a mouse but we have shown this is still a problem. In all there is much website owners can do to make the use of mobile web easier but it may often require a different mindset to desktop, but also not flipping the other way entirely and forgetting desktop users either.

Hope that suffices but feel free to change if you'd rather say something else.